shared/src/simulation/behaviors/manage-patient-transport-to-hospital.ts

```diff
@@ -0,0 +1,513 @@
+                                Object.values(patientsExpectedInRegions).some(
+                                    (patientCounts) =>
+                                        patientCounts[category] > 0
+                                )
+                            );
+
+                        if (
+                            !highestCategoryThatIsNeeded ||
+                            orderedPatientCategories.indexOf(
+                                highestCategoryThatIsNeeded
+                            ) >
+                                orderedPatientCategories.indexOf(
+                                    behaviorState.maximumCategoryToTransport
+                                )
+                        ) {
+                            break;
+                        }
+
+                        const simulatedRegionIdWithBiggestNeed = Object.entries(
+                            patientsExpectedInRegions
+                        ).sort(
+                            ([, patientCountsA], [, patientCountsB]) =>
+                                patientCountsB[highestCategoryThatIsNeeded] -
+                                patientCountsA[highestCategoryThatIsNeeded]
+                        )[0]![0];
+
+                        const vehicleType = getNextVehicleForPatientStatus(
+                            behaviorState,
+                            highestCategoryThatIsNeeded
+                        );
+
+                        const targetTransferPoint = getElementByPredicate(
+                            draftState,
+                            'transferPoint',
+                            (transferPoint) =>
+                                isInSpecificSimulatedRegion(
+                                    transferPoint,
+                                    simulatedRegionIdWithBiggestNeed
+                                )
+                        );
+
+                        if (vehicleType) {
+                            addActivity(
+                                simulatedRegion,
+                                SendRemoteEventActivityState.create(
+                                    nextUUID(draftState),
+                                    behaviorState.requestTargetId,
+                                    TransferVehiclesRequestEvent.create(
+                                        { vehicleType: 1 },
+                                        'transferPoint',
+                                        targetTransferPoint.id,
+                                        simulatedRegion.id,
+                                        undefined,
+                                        PatientTransferOccupation.create(
+                                            simulatedRegion.id
+                                        )
+                                    )
+                                )
+                            );
+                        }
+                    }
+                    break;
```

Comment:
Pleas emove the `break;` statements (here and at the other `case`s) inside of the braces.